packages/compass-e2e-tests/helpers/commands/wait-for-connection-screen.ts

```diff
@@ -1,9 +1,14 @@
+import { TEST_COMPASS_WEB } from '../compass';
 import type { CompassBrowser } from '../compass-browser';
 import * as Selectors from '../selectors';
 
 export async function waitForConnectionScreen(
   browser: CompassBrowser
 ): Promise<void> {
-  const connectScreenElement = await browser.$(Selectors.ConnectSection);
+  // TODO: can we not just make this the same selector in both cases somehow?
```

Comment:
Good point, we might be able to use connection-form in the sandbox, this should make this stuff more unified even if connection form will not end up in data explorer